test/__snapshots__/index.spec.ts.snap

```diff
@@ -679,8 +679,8 @@ Array [
 exports[`import rules import/no-default-export fails on an invalid fixture 1`] = `
 Array [
   Object {
-    "column": 1,
-    "endColumn": 20,
+    "column": 8,
```

Comment:
I assume the change is related to this update (taken from release notes): https://github.com/import-js/eslint-plugin-import/pull/2299